components/scanners/cdxgen/README.md

```diff
@@ -0,0 +1,17 @@
+# semgrep
```

Comment:
Should this be updated to reflect what the component is and does?

Reply:
➕